.github/workflows/kbs-e2e.yaml

```diff
@@ -49,6 +49,12 @@ jobs:
           target/
         key: rust-${{ hashFiles('./Cargo.lock') }}
 
+    - name: Set up SGX/TDX certificates cache
+      uses: actions/cache@v4
+      with:
+        path: /root/.dcap-qcnl
```

Comment:
I don't know if this breaks but I was wondering if this is more future proof?
```suggestion
        path: $HOME/.dcap-qcnl
```

Reply:
do you think the path statement in that github actions works with envs?

Reply:
I'm not sure. I'm fine with `/root` as well

Reply:
@mkulke sorry, for my ignorance, who/how /root/.dcap-qcnl was generated? Is it  manually set on the `self-hosted` runner? All in all, I didn't understand why it is cached.

Reply:
@wainersm it's a cache for the SGX/TDX TCB data written by `libsgx-dcap-quote-verify`.The reasons for why are explained in #329

Reply:
ah sorry, the PR description was not updated.